inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfextract/PDFExtractor.java

```diff
@@ -345,7 +345,7 @@ else if (font instanceof PDType0Font) {
         try {
             spaceWidthText = font.getSpaceWidth() * glyphSpaceToTextSpaceFactor;
         }
-        catch (Throwable e) {
+        catch (Exception e) {
```

Comment:
It is external code, but we kind of adopted it because the upstream project (https://github.com/yuichiro-s/pdfextract) has been abandoned. So in this case, I'd say the change is in fact an improvement.

Reply:
Catching Throwable is intentional here as we want to continue the execution even if a particular recommender fails.